docs/management_api.yml

```diff
@@ -386,6 +387,52 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /deployments/{deployment_id}/devices/list:
+    get:
+      operationId: Devices List in Deployment
```

Comment:
The operationId is very similar to the one we deprecated (List Devices in Deployment vs. Devices List in Deployment). As we render this in the API docs as menu item in the sidebar, I would suggest improving the name. For example, "List Devices in Deployment with pagination". We'll remove the "with pagination" as soon as we remove the deprecated EP.